app/controllers/items/partial_move.js

```diff
@@ -32,6 +35,19 @@ export default Ember.Controller.extend({
       }else{
         this.transitionToRoute('items.search_location', item.id, { queryParams: { isPartialMove: true }});
       }
+    },
+    editPackagesLocation(packagesLocation){
+      this.set('isEditing', true);
+      packagesLocation.set('isEditing', true);
+    },
+
+    notNow(item){
+      var editedPackagesLocations = item.get('packagesLocations').filterBy('isEditing', true);
+      editedPackagesLocations.forEach(packagesLocation => {
```

Comment:
Can we rename ** packagesLocation** to **packageLocation** as it is single record?